examples/microprofile/telemetry/secondary/src/main/resources/META-INF/microprofile-config.properties

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright (c) 2013 Oracle and/or its affiliates.
```

Comment:
I believe the builds will fail if the copyright is not correct. Check each file.